conan/api/subapi/cache.py

```diff
@@ -77,6 +80,69 @@ def check_integrity(self, package_list):
         checker = IntegrityChecker(cache)
         checker.check(package_list)
 
+    def sign(self, package_list):
+        """Sign packages with the package signing plugin"""
+        cache = PkgCache(self._conan_api.cache_folder, self._api_helpers.global_conf)
+        pkg_signer = PkgSignaturesPlugin(cache, self._conan_api.home_folder)
+        if not pkg_signer.is_configured:
```

Comment:
This doesn't seem correct, the only thing that is needed for this ``sign()`` is the verify function, but ``is_configured`` requires both functions to be define.